package-lock.json

```diff
@@ -10359,6 +10359,27 @@
             "process": "~0.5.1"
           }
         },
+        "https-proxy-agent": {
+          "version": "3.0.0",
+          "resolved": "https://registry.npmjs.org/https-proxy-agent/-/https-proxy-agent-2.2.1.tgz",
```

Comment:
Confused why the resolve here is still 2.2.1, surely this means the vulnerability isn't fixed? 

cc @hindsc52 @jamesdonoh

Reply:
Tried `npm install --save https-proxy-agent@3.0.0` and to check and this should be... 
`"resolved": "https://registry.npmjs.org/https-proxy-agent/-/https-proxy-agent-3.0.0.tgz",`